Sources/Plasma/PubUtilLib/plMessage/CMakeLists.txt

```diff
@@ -150,6 +152,7 @@ target_link_libraries(plMessage
         plGImage
         plInputCore # plInputIfaceMgrMsg
         plNetGameLib
+        plSDL # plSDLModifierStateMsg
```

Comment:
Should be `PUBLIC` since `plSDL/plSDL.h` is included in a public header.

Reply:
It is not included in the header, only in the implementation

Reply:
Whoops, looking at wrong filename.